.github/workflows/pypi-release.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
 
   upload-to-pypi:
     needs: test-built-dist
-    if: github.event_name == 'release' && startsWith(github.event.ref, 'refs/tags/v')
+    if: github.event_name == 'release' && startsWith(github.ref, 'refs/tags/v')
```

Comment:
do we even need the extra condition? `event_name == 'release'` should make that redundant.

Reply:
As long as we are okay with publishing every tag to PyPI, then I think the `event_name` is redundant...

Reply:
no, I meant to remove the `startsWith(...)` part: `github.event_name == 'release'` should guarantee that someone created a release through the web page

Reply:
I see... Yes, that's right. We don't need that part.

Reply:
Thanks for catching that!